pugjs/engine.go

```diff
@@ -98,16 +99,27 @@ func init() {
 }
 
 // Inject injects the EventSubscibers dependencies
-func (e *EventSubscriber) Inject(engine *Engine, logger flamingo.Logger) {
+func (e *EventSubscriber) Inject(engine *Engine, logger flamingo.Logger, startup *Startup) {
 	e.engine = engine
 	e.logger = logger
+	e.startup = startup
 }
 
-// Notify the event subscriper
+// Notify the event subscriber
 func (e *EventSubscriber) Notify(_ context.Context, event flamingo.Event) {
-	if _, ok := event.(*flamingo.StartupEvent); ok {
+	switch ev := event.(type) {
+	case *flamingo.StartupEvent:
```

Comment:
Unsure if we need this at all. This will load the templates for the root area as the injections on startup are on the root level. 
But: if the root area is not routed, we may not need the templates here; if it is routed, the templates will be loaded by the `AreaRoutedEvent` below.
Of course, a double loading will be blocked by the mechanisms inside LoadTemplates, so an additional call would not cause too much harm.